microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -279,13 +277,9 @@ To learn more about which API methods are available and to view example code, se
 
 To provide feedback about the Ad Selection API origin trial:
 
-1. Go to the [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/) portal at Github.io.
-
-1. Go to the row for the **Ad Selection API** origin trial.
-
-1. Click the **issues** button in the row.  The `privacy-preserving-ads` repo opens.
+1. Go to the [MSEdgeExplainers](https://github.com/MicrosoftEdge/MSEdgeExplainers) repo at GitHub.com.
 
-1. Submit a new issue, or click the **Issues** tab to view the issues.
+1. Submit a new issue, or click the **Issues** tab to view the existing issues.
 
 
 <!-- ====================================================================== -->
```

Comment:
probably delete line 308-310 in See Also section:
```
Origin trials:
* [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/)
```
https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/